CDP4IMEInstaller.Tests/DevExpress.wxs.liquid

```diff
@@ -13,8 +12,8 @@
       {%- assign specialChar2 = "}" -%}
 
       {%- for wxsObject in KnownAssemblies.WxsObjects -%}
-      <Component Id="{{wxsObject.ComponentId}}" Guid="{{specialChar1}}{{wxsObject.Guid}}{{specialChar2}}" Win64="$(var.Win64)">
-        <File Id="{{wxsObject.FileId}}" Vital="yes" Source="$(var.CDP4-COMET-CE.TargetDir){{wxsObject.AssemblyName}}" KeyPath="yes" Assembly=".net"></File>
+      <Component Id="{{wxsObject.ComponentId}}" Guid="{{specialChar1}}{{wxsObject.Guid}}{{specialChar2}}" >
+        <File Id="{{wxsObject.FileId}}" Vital="yes" Source="$(var.CDP4COMETCE.TargetDir){{wxsObject.AssemblyName}}" KeyPath="yes" Assembly=".net"></File>
```

Comment:
`CDP4-COMET-CE` does not work anymore?

Reply:
With WiX 4, Identifier cannot contain '-' anymore